### ISSUE TITLE
Midpoint review - Content Feedback - Authenticated Experience, My VA, Audit UX Improvements

### ISSUE THREAD

**Issue by CherylEvans** [department-of-veterans-affairs/va.gov-team#48624]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
- Great job so far!

## Feedback
- Practice areas will document their feedback following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback).

**Must:**

- [LOA1 state](https://www.sketch.com/s/9b0e6efc-423a-4354-9db3-ab2083d566c9/a/padQ1OG) I recommend using an em dash as opposed to a plain dash.  VA Guidance with regard to punctuation can be found [here](https://design.va.gov/content-style-guide/punctuation#em-and-en-dashes-hyphensurl).

-- Recommend using "We need to make sure you're you—and not someone pretending to be you—before we can give you access to your personal and health-related information.", as opposed to, "We need to make sure you're you - and not someone pretending to be you - before we can give you access to your personal and health-related information."

-- According to AP Stylebook, 56th edition, an em dash is used to signal abrupt change within a phrase.

- [Error states](https://www.sketch.com/s/9b0e6efc-423a-4354-9db3-ab2083d566c9/a/uuid/36FD1008-AA20-4E81-BCAF-FFBAE0F4070F) With regard to verbiage, in order to be consistent with the Health care section, Claims and appeals should read, "We can't access **your** claims or appeals information right now."  The Health care section reads, "We can't access your appointment information."  This is in accordance with the [VA plain language standards](https://design.va.gov/content-style-guide/content-principles/plain-language).

- [Error states](https://www.sketch.com/s/9b0e6efc-423a-4354-9db3-ab2083d566c9/a/uuid/36FD1008-AA20-4E81-BCAF-FFBAE0F4070F) With regard to verbiage, in order to be consistent with the Claims and appeals section, Health care section should read, "We can't access your appointment information **right now**."

**Should:**

**Consider:**

## Platform directions
- Remove the governance-team label
- Update "Issue Title"
- Link to collab cycle Request epic
- Add your feedback
- Remove the CC-Request label (only if you created the ticket using the New Issue link from the Collaboration Cycle Reviews board)
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review QA Only**: VFS PM, yourself (optional)

**Comment by ajakabcin** [comment]
Hello! We'll be making the following updates which address the "must" feedback issued at midpoint review as well as some feedback given during our content review:

1. Updating all instances of "view" to "review"
2. Capitalizing GI Bill and GI Statement of Benefits
3. Updating Claims and Appeals error state: update copy to “We can't access your claims or appeals information right now” to be consistent with error on appointments in Health care section: “We can’t access your appointment information”
4. Updating Health care error state: update copy of appointments error message to read “We can’t access your appointment information right now” (adds phrase “right now” to match claims and appeals”

That work can be tracked in #48701 , #48704 , #48705 and #47523

**Comment by ajakabcin** [comment]
Hi @shiragoodman - We'll be making the following updates which address the "must" feedback issued at midpoint review as well as some feedback given during our content review:

1. Updating all instances of "view" to "review"
2. Capitalizing GI Bill and GI Statement of Benefits
3. Updating Claims and Appeals error state: update copy to “We can't access your claims or appeals information right now” to be consistent with error on appointments in Health care section: “We can’t access your appointment information”
4. Updating Health care error state: update copy of appointments error message to read “We can’t access your appointment information right now” (adds phrase “right now” to match claims and appeals”

That work can be tracked in #48701 , #48704 , #48705 and #47523 

cc @Samara-Strauss @ericasauv09 @aagosto90

**Comment by ajakabcin** [comment]
resolved - addressed in #49558, #48701 , #48704 , #48705 and #47523
